db/migrations/20221212181152_stream_events.sql

```diff
@@ -0,0 +1,23 @@
+--FIXME: run `goose fix`
+
+-- +goose Up
+CREATE TYPE livestream_event_type AS ENUM (
+    'started',
+    'ended',
+    'linked',
+    'unlinked',
+    'stream_received',
+    'stream_lost',
```

Comment:
Would say todo kebab case to match other status', but I'm not particularly fussed